### PR TITLE
ADEN-11170 Implementing UAP:JWPlayer feature

### PR DESCRIPTION
**Pull request by dmnsobczak**
Ticket: https://wikia-inc.atlassian.net/browse/ADEN-11170

### CODE REVIEW

File: src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { context, utils } from '@ad-engine/core';
+import { universalAdPackage } from '../../../templates/uap/universal-ad-package';
```
Comment (preludek) [inline src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts:2]: I think this import can be shortened to `import { universalAdPackage } from '../../../templates';`

File: src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+	get(): Promise<void> {
+		if (!this.isEnabled()) {
+			return Promise.resolve();
+		}
+
+		return this.getExtendedPromise();
+	}
+
+	resolve(lineItemId: string = null, creativeId: string = null): void {
```
Comment (preludek) [inline src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts:19]: Is TypeScript ok with assigning null to string? :D

**Review by preludek — APPROVED** [review]

File: src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts]: Actually `undefined` is also false, so I removed default values :D

File: src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-products/video/jwplayer/helpers/jwplayer-inhibitor.ts]: Done!

**Pull request merged**